### PR TITLE
Add calico-crd-installer

### PR DESCRIPTION
**Pull request by tfussell**
See description https://github.com/giantswarm/crd-installer/pull/1 for more info.

### CODE REVIEW

File: images.yaml [hunk 1]
```diff
@@ -583,6 +583,24 @@
...
   patterns:
   - pattern: '>= v3.9.5'
+- name: quay.io/calico/kube-controllers
+  overrideRepoName: calico-crd-installer
+  patterns:
+  - pattern: '>= v3.16.0'
+    customImages:
+    - dockerfileOptions:
+      - |
+        FROM quay.io/giantswarm/crd-installer:0.0.0-add8f5797322a6e44489740b17a59c0a64ae36e7 AS installer
```
Comment (tfussell) [inline images.yaml:593]: Will replace with 0.0.1 once this PR https://github.com/giantswarm/crd-installer/pull/1 is released.

File: images.yaml [hunk 2]
```diff
@@ -583,6 +583,24 @@
...
+        FROM quay.io/giantswarm/alpine:3.13.5 AS downloader
+        WORKDIR /tmp/crd-installer
+        COPY --from=0 /usr/bin/kube-controllers /crd-installer/kube-controllers
+        COPY --from=installer /scripts /scripts
+        RUN CALICO_VERSION=$(/crd-installer/kube-controllers -version) && \
+            /scripts/download-calico-crds.sh $CALICO_VERSION
```
Comment (MarcelMue) [inline images.yaml:599]: Okay scratch my question in the other review. Should have checked there here first.

**Review by MarcelMue — APPROVED** [review]
LGTM for a first iteration

**Comment by tfussell** [comment]
I'm going to try creating a cluster with it today, but I need the image to exist so I'll merge this for now and make changes as needed.

**Pull request merged**